core/client/fs/src/main/java/alluxio/client/file/cache/CacheEvictor.java

```diff
@@ -19,11 +23,12 @@
 public interface CacheEvictor {
 
   /**
+   * @param conf the alluxio configuration
    * @return a CacheEvictor instance
    */
-  static CacheEvictor create() {
-    // return corresponding CacheEvictor impl
-    return new LRUCacheEvictor();
+  static CacheEvictor create(AlluxioConfiguration conf) {
+    return CommonUtils.createNewClassInstance(
+        conf.getClass(PropertyKey.USER_CLIENT_CACHE_EVICTOR_CLASS), new Class[0], new Object[0]);
```

Comment:
I think you want `null` instead of empty arrays?

Reply:
Updated.